src/components/table/TimeSeriesTable.vue

```diff
@@ -142,6 +145,8 @@ const props = withDefaults(defineProps<Props>(), {
 const emit = defineEmits(['change'])
 
 const store = useFewsPropertiesStore()
+const configStore = useConfigStore()
+const readOnlyMode = ref<boolean>(configStore.general.readonlyMode ?? false)
```

Comment:
readonlyMode still needs to be added to the configStore

Reply:
Can remove extend due to update

Reply:
![image](https://github.com/Deltares/fews-web-oc/assets/144685504/1587a645-0375-4a87-9fa9-17421baef364)
It is there since the update